lighthouse-cli/test/smokehouse/test-definitions/tricky-metrics/expectations.js

```diff
@@ -44,6 +44,18 @@ module.exports = [
       },
     },
   },
+  {
+    lhr: {
+      requestedUrl: 'http://localhost:10200/delayed-lcp.html',
+      finalUrl: 'http://localhost:10200/delayed-lcp.html',
+      audits: {
+        'largest-contentful-paint': {
+          // LCP is after the ~7s XHR and the ~7s image.
+          numericValue: '>13500',
```

Comment:
Why isn't this strictly >14s?

Reply:
paranoia :) 

IIRC this habit of mine came from a time when our time origin was less reliable that caused flakes of up to a few 100 ms.  14s should be fine these days.
```suggestion
          numericValue: '>14000',
```